fastlane/lib/fastlane/actions/update_app_identifier.rb

```diff
@@ -47,6 +47,16 @@ def self.run(params)
         end
       end
 
+      def self.resolve_path(path, xcodeproj_path)
+        project_dir = File.dirname(xcodeproj_path)
+        # SRCROOT and PROJECT_DIR are the same
+        %w{SRCROOT PROJECT_DIR}.each do |variable_name|
+          path = path.sub("$(#{variable_name})", project_dir)
+        end
+        path = File.join(project_dir, path) unless path.start_with?('/')
```

Comment:
Should you use `File.absolute_path` instead?

```ruby
File.absolute_path(path, project_dir)

File.absolute_path("/aaa/bbb", project_dir) #-> /aaa/bbb
File.absolute_path("aaa/bbb", project_dir) #-> /path/to/project/aaa/bbb

```

Reply:
`absolute_path` does indeed makes things clearer. Fixed